Octokit/Models/Request/RepositoryForksListRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Octokit.Internal;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class RepositoryForksListRequest : RequestParameters
+    {
+        public RepositoryForksListRequest()
+        {
+            Sort = Sort.Newest; // Default in accordance with the documentation
+        }
+
+        [Parameter(Key = "sort")]
+        public Sort Sort { get; set; }
+
+        internal string DebuggerDisplay
+        {
+            get
+            {
+                return String.Format(CultureInfo.InvariantCulture, "Sort: {0}", Sort);
+            }
+        }
+    }
+
+    public enum Sort
+    {
+        [Parameter(Value = "newest")]
```

Comment:
Inside our serializer we'll take the C# values and lowercase them, so these overrides aren't really necessary.

Where they do become helpful is where you want to show a completely different value.

Feel free to drop them.